### PR TITLE
Feature/add translation function

### LINKED ISSUE #9: Setup translations / i18n-things

### PR DESCRIPTION
**Pull request by eevajonnapanula**
Resolves #9 

This PR adds translations setup, and adds Finnish and Swedish versions of the site, but is missing translations to these languages. 

The structure of the site will change a little: 
- The English site is still at the `/`, but english events will have the `/en/event-name`-url. 
- The Finnish site will be at `/fi` and Swedish site at `/sv`, and their events will have the same url structure (so, `/lang-code/event-name`)

I did not add any possibility to change the language yet, as there are so many translations missing. I will create an issue about this.

### CODE REVIEW

**Review by magdapoppins — COMMENTED** [review]
Looks good, sorry for leaving this pending for so long! Added some thoughts about instructions in regard to creating the translations. 

Question 1: I don't know how much harder or easier it would be, but as a thought I think it would make more sense to have all translations in the same `.md` document. I wonder if it would be possible to have for example one `events/our-event.md` document with sections for the different languages and the same metadata for all? Would it be possible to filter the content under specific titles? E.g.:
```
---
(metadata)
---
En
---
Today we had a fine robotics club.

Fi
---
Robotiikkakerho kokoontui tänään.
```

Question/comment 2: Since a wild event has appeared since this, would need to rebase unto dev and make these same changes for it.

File: docs/adding-new-events.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+---
+```
+
+Some notes:
+
+- **title** is a string
+- **date** in "YYYY-MM-DD"-format
+- **speakers** as a list
+- **host** is a string
+- **lang** is either en, fi or sv
```
[the remark this answers is not in the document]
Comment (lgrigoryeva1) [reply docs/adding-new-events.md:29]: @magdapoppins I don't think there's a need for it, Fi/Swe/Eng seem enough to me :1st_place_medal:

File: docs/translations.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+```json
+"keyname": {
+  "en": "English text",
+  "fi": "Finnish text",
+  "sv": "Swedish text"
+}
+```
+
+where `keyname` is logical, for example text for the frontpage would start with `frontpage.`. (see other keys in the file). You have to provide at least the english version, but others are recommended as well. If you don't know the translation, add at least the english version to every key.
```
[the remark this answers is not in the document]
Comment (eevajonnapanula) [reply docs/translations.md:35]: Yes, good point, will do that 👍

File: docs/adding-new-events.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Adding a new event
+
+When you are adding a new event, here's the procedure: 
+
+1. Create a folder with the name of the event (see other events as example) under the correct language folder
```
[the remark this answers is not in the document]
Comment (eevajonnapanula) [reply docs/adding-new-events.md:5]: This sounds like a good plan. Will add that

**Comment by eevajonnapanula** [comment]
Answer to the question 1: It is possible, but would need some wrangling to do, and honestly, I'm not sure if I have time to do that (before the meetup next weekend) 😅  so I'd leave it that way for now, as this way many things are generated automatically. We could create an issue about this change with some instructions and goals (I have some ideas how this could be done)

Question 2: Yes! I will rebase that and do the edits

**Review by magdapoppins — APPROVED** [review]
🚀 !

**Pull request merged**